ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/ProductionCoordinatorFreezerBurn.asset

```diff
@@ -0,0 +1,24 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: d8cfffae36be18b40b59e8e84d47a6ad, type: 3}
+  m_Name: ProductionCoordinatorFreezerBurn
+  m_EditorClassIdentifier: 
+  company: {fileID: 11400000, guid: d0ddb5a02bb3d9a4f86d3ef8fd732359, type: 2}
+  positionName: Production Coordinator
+  jobSummaryLines:
+  - Our company is looking for someone willing to take a lead in our production.
```

Comment:
either "take **the** lead **on** our production **team**" or "take a lead**ing role** in our production **team**"